zebra/rt_netlink.c

```diff
@@ -4502,12 +4529,23 @@ static int netlink_ipneigh_change(struct nlmsghdr *h, int len, ns_id_t ns_id)
 	int l2_len = 0;
 	enum dplane_op_e op;
 
+	if (!h) {
```

Comment:
how is this a problem but all the other places where we pass around a `struct nlmsghdr *h` not a problem?  I would far far preferr asserting that h is not null somewhere up the call chain where the struct nlmsghdr pointer is found instead of having to touch a million spots in the code